src/components/fields/password-field/password-field.spec.js

```diff
@@ -191,3 +191,24 @@ describe('when field is touched and has errors', () => {
     });
   });
 });
+
+describe('when input is untouched', () => {
```

Comment:
from the code I got the impression that it's disabled if the value is not there, not if it's untouched.

Reply:
Yes, I tested the value but wrote the wrong description.